src/osd/OSDMap.cc

```diff
@@ -4448,21 +4448,21 @@ void OSDMap::check_health(health_check_map_t *checks) const
     if (br < nr) {
       ostringstream ss;
       ss << "backfillfull_ratio (" << br
-	 << ") < nearfull_ratio (" << nr << "), increased";
+	 << ") < nearfull_ratio (" << nr << "), should increase backfillfull_ratio";
```

Comment:
i think current error message is correct and accurate. "increased" is what the osd is doing on be half of user when facing a misconfigured OSD. and user should be aware of this change. hence it's reported as an "error".

Reply:
@tchaikov ok, I am fine to close this PR, thanks.

@xiexingguo thank you all the same.